docs/examples/README.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Miscellaneous Dafny Examples
+---
+
+The links below connect to annotated examples of Dafny source code,
+demonstrating various coding techniques or proof capabilities..
+Each of these prove successfully.
+
+The files for these examples can be found in 
+the [examples folder of Dafny's github project](https://github.com/dafny-lang/dafny/tree/master/docs/examples).
+
+- [Simple Maximum method](https://github.com/dafny-lang/dafny/tree/master/Test/examples/maximum.dfy)
+- [Tutorial on parser combinators](https://github.com/dafny-lang/dafny/tree/master/Test/examples/parser_combinators.dfy)
+- A simple compiler: [code](https://github.com/dafny-lang/dafny/tree/master/Test/examples/Simple_compiler/Compiler.dfy) and [discussion](../../Test/examples/Simple_compiler/README)
```

Comment:
Consider just linking to the README, since it provides the context and names `Compiler.dfy` itself anyway.

Reply:
Done